src/screens/IdentityBackup.js

```diff
@@ -34,9 +34,11 @@ import { withAccountStore } from '../util/HOC';
 import testIDs from '../../e2e/testIDs';
 import ScreenHeading from '../components/ScreenHeading';
 import { alertBackupDone, alertCopyBackupPhrase } from '../util/alertUtils';
+import Button from '../components/Button';
 
 function IdentityBackup({ navigation, accounts }) {
 	const [seedPhrase, setSeedPhrase] = useState('');
+	const [wordsNumber, setWordsNumber] = useState(12);
```

Comment:
I hate to be so nitty but I think the default should be 24 words, and power users can opt to use just a slightly less safe 12 word mnemonic. Just my opinion, not sure what Gav asked for here.

Reply:
true, the original discussion was using 12 words as an option.